data/migrations/015-phone_numbers.js

```diff
@@ -8,7 +8,7 @@ exports.up = function (knex) {
       .references('recipient_id')
       .inTable('recipients')
       .onUpdate('CASCADE')
-      .onDelete('RESTRICT');
+      .onDelete('CASCADE');
```

Comment:
Might want to confirm that we want a delete to cascade throughout the app upon deleting a recipient. Possibly with stakeholder?